### PR TITLE
BZ2066913: fix RAID data volume docs

### PR DESCRIPTION
**Pull request by jldohmann**
OCP Version: 4.8+

Redo of #44061 because I messed up with git 🙃 

Direct doc preview link: https://deploy-preview-44188--osdocs.netlify.app/openshift-enterprise/latest/installing/install_config/installing-customizing.html#installation-special-config-raid_installing-customizing

### CODE REVIEW

**Comment by jldohmann** [comment]
@bgilbert could you PTAL again for SME ack? I tweaked the second step, and removed the third step that contains the `oc create` command.

This change is meant for 4.8+. Looking at the 4.7 and 4.6 docs, I believe the only change need [based on your feedback ](https://github.com/openshift/openshift-docs/pull/44061#pullrequestreview-929606335)is just a removal of the second step:

- [4.7 PR](https://github.com/openshift/openshift-docs/pull/44211)
- [4.6 PR](https://github.com/openshift/openshift-docs/pull/44213)

Like you suggested, I'll write up a new procedure for documenting day 2 changes

**Comment by jldohmann** [comment]
4bc768b...6076459 add prerequisites section

File: modules/installation-special-config-raid.adoc [hunk 1]
```diff
@@ -87,19 +99,12 @@ storage:
...
+. Create a RAID manifest from the Butane config you created in the previous step and save it to the `<installation_directory>/openshift` directory. For example, to create a manifest for the compute nodes, run the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <manifest_name>.yaml
+$ butane $HOME/clusterconfig/<butane_config>.bu -o <installation_directory>/openshift/<manifest_name>.yaml <1>
 ----
+<1> Replace `<butane_config>` and `<manifest_name>` with the file names from the previous step. For example, `raid1-alt-storage.bu` and `raid1-alt-storage.yaml` for secondary disks.
 
 . Save the Butane config in case you need to update the manifest in the future.
```
Comment (bgilbert) [inline modules/installation-special-config-raid.adoc:110]: In some similar procedures we say e.g. `Continue with the remainder of the {product-title} installation.` to help make it clear that this needs to be done at installation time.

**Review by bgilbert — APPROVED** [review]
LGTM

**Review by adellape — APPROVED** [review]

**Pull request merged**